scripts/Packages/Package_BGA/bga.yaml

```diff
@@ -345,3 +345,14 @@ WLP-9_1.448x1.468mm_Layout3x3_P0.4mm_Ball0.27mm_Pad0.25mm:
   paste_margin: 0.000001
   layout_x: 3
   layout_y: 3
+Dialog_VFBGA-86_6x6mm_Layout10x10_P0.55mm_Ball0.25mm_Pad0.2mm:
```

Comment:
Ball diameter is in the name, but not set as a property.

Reply:
It's not really clear for me. What do you mean? I looked at the previous part (WLP-9_1.448x1.468mm_Layout3x3_P0.4mm_Ball0.27mm_Pad0.25mm) and made similar.

Reply:
I was stupid, misread a line, made a stupid assumption and wrote that comment. Ignore it.